CI/setup-windows-sdk.sh

```diff
@@ -144,12 +149,13 @@ fi
 
 pacman_attempts=1
 while true; do
-  if /usr/bin/pacman -Su --needed --noconfirm \
+  if /usr/bin/pacman -S --needed --noconfirm \
     git \
     man \
     rsync \
-    "mingw-w64-${BUILDCOMPONENT}-ccache" \
     "mingw-w64-${BUILDCOMPONENT}-toolchain" \
+    "mingw-w64-${BUILDCOMPONENT}-gcc-compat" \
```

Comment:
This package is not available for `MINGW32`/`MINGW64` (or `UCRT64`) it is only for the `CLANG32`/`CLANG64` (or `CLANGARM64`) cases - so this will not work for the 32-Bit Windows (`MINGW32`) builds of Mudlet...

Reply:
Yea again this PR not intended to provide a solution, just running experimental builds using the CI system. It is ugly/dirty/does dumb things.